rivery_cli/rivery_session.py

```diff
@@ -1,9 +1,12 @@
+import logging
+import time
 import zlib
+from itertools import chain, repeat
 
+import click
 import requests
+
 from .utils import bson_utils as json_util, utils
-import logging
-import simplejson as json
```

Comment:
Why dtoppong simplejson?

Reply:
wasn't in use (CTRL ALT O did it actually :) )